backend/cmd/headlamp.go

```diff
@@ -75,6 +76,7 @@ const (
 
 const isWindows = runtime.GOOS == "windows"
 
+// PortForward Information forwarded by Port forwarding
```

Comment:
I'm not sure about this one:
> PortForward Information forwarded by Port forwarding

How about something like this instead?
> PortForward Information for internal use for each port that is being forwarded.

It adds that it is an internal structure of the ports being forwarded.

Reply:
OK